static/operator.yaml

```diff
@@ -203,14 +217,14 @@ spec:
       serviceAccountName: istio-operator
       containers:
         - name: istio-operator
-          image: gcr.io/istio-testing/operator:master-latest-daily
+          image: gcr.io/istio-testing/operator:1.4-dev
```

Comment:
is there a post install job for this that you added?  If not, lmk and I'll add one.